lib/core/utils/get-standards.js

```diff
@@ -0,0 +1,6 @@
+import standards from '../../standards/index'
```

Comment:
```suggestion
import standards from '../../standards'
```

Reply:
Is this worth requesting changes for? There's no functional difference between the two.

Reply:
Guess it just code style at this point. No where else do we ask for `/index` and I thought when we added webpack we purposefully removed `index` as webpack handled it.

Reply:
I see. IMO style decisions shouldn't result in requested changes. Instead we should have Prettier/ESLint handle that sort of thing for us, otherwise we end up painting the bike shed 14 different colors.

NBD tho 🤷‍♂️

Reply:
I'm not sure there is a prettier or eslint config option to remove `/index` from the end of things. Looks like if we want to enforce it we'll probably need to make our own eslint rule for it.

Reply:
Elsewhere we've [just documented that we don't use "index" files](https://github.com/dequelabs/walnut/tree/develop/app#patterns--rules) (they're confusing- which index do I have open/hard to search for in editors/prone to nitpicks like this/etc):

> `index.*` files should not be used; `index.html` is the exception


If we feel strongly enough about not `import`ing files like this, we could document it (maybe in the contribution guidelines) rather than writing a custom lint rule for is.

Reply:
I agree, indexes are messy. I'll update the PR, and I'm going to open another one to get rid of the index files.